devices/lixee.js

```diff
@@ -0,0 +1,475 @@
+                'BBRHCJW',
+                'BBRHPJW',
+                'BBRHCJR',
+                'BBRHPJR',
+                'DEMAIN',
+                'PEJP'].includes(a.exposes.name));
+            break;
+        case 'HC..':
+            myExpose = myExpose.filter((a) => ![
+                'BASE',
+                'EJPHN',
+                'EJPHPM',
+                'BBRHCJB',
+                'BBRHPJB',
+                'BBRHCJW',
+                'BBRHPJW',
+                'BBRHCJR',
+                'BBRHPJR',
+                'DEMAIN',
+                'PEJP'].includes(a.exposes.name));
+            break;
+        case 'EJP.':
+            myExpose = myExpose.filter((a) => ![
+                'BASE',
+                'HCHC',
+                'HCHP',
+                'BBRHCJB',
+                'BBRHPJB',
+                'BBRHCJW',
+                'BBRHPJW',
+                'BBRHCJR',
+                'BBRHPJR',
+                'DEMAIN'].includes(a.exposes.name));
+            break;
+        case currentTarf.startsWith('BBR'):
+            myExpose = myExpose.filter((a) => ![
+                'BASE',
+                'HCHC',
+                'HCHP',
+                'EJPHN',
+                'EJPHPM',
+                'PEJP'].includes(a.exposes.name));
+            break;
+        default:
+            break;
+        }
+    }
+
+    return myExpose;
+}
+
+const definition = {
+    zigbeeModel: ['ZLinky_TIC'],
+    model: 'ZLinky_TIC',
+    vendor: 'LiXee',
+    description: 'Lixee ZLinky',
+    fromZigbee: [seMeteringFZ, haMeterIdentificationFZ, haElectricalMeasurementFZ, seLixeePrivateFZ],
+    toZigbee: [],
+    exposes: (device, options) => {
+        return getCurrentConfig(device, options)
+            .map((e) => e.exposes);
+    },
+    options: [
+        exposes.options.measurement_poll_interval(),
+        exposes.enum(`linky_mode`, ea.SET, [linkyModeDef.legacy, linkyModeDef.standard])
+            .withDescription(`Counter with TIC in mode standard or historique. Requires re-configuration (default: historique)`),
+        exposes.enum(`energy_phase`, ea.SET, [linkyPhaseDef.single, linkyPhaseDef.three])
+            .withDescription(`Power with single or three phase. Requires re-configuration (default: single_phase)`),
+        exposes.binary(`production`, ea.SET, true, false).withDescription(`If you produce energy back to the grid. Requires re-configuration (only linky_mode: ${linkyModeDef.standard}, default: false)`),
+    ],
+    configure: async (device, coordinatorEndpoint, logger) => {
+        const SW1_ENDPOINT = 1;
+        const endpoint = device.getEndpoint(SW1_ENDPOINT);
+
+        await reporting.bind(endpoint, coordinatorEndpoint, [
+            clustersDef._0x0702, /* seMetering */
+            clustersDef._0x0B01, /* haMeterIdentification */
+            clustersDef._0x0B04, /* haElectricalMeasurement */
+            clustersDef._0xFF66, /* liXeePrivate */
+        ])
+            .catch((e) => {
```

Comment:
Please don't catch the error, otherwise z2m will not attempt to reconfigure it if configure fails (It think it successfully configured the devices if you suppress errors)